From 14f15cb0f03defa8efb4c8e2fece58e50655be6b Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Sun, 23 Jul 2017 22:20:29 -0400
Subject: [PATCH] src/xmlrpc_server_abyss.c: fix formatting issues

Fixed when compile with "-Wformat -Wformat-security -Werror=format-security":
|src/xmlrpc_server_abyss.c:771:13: error: format not a string literal
and no format arguments [-Werror=format-security]
|             xmlrpc_faultf(envP, error);
|             ^~~~~~~~~~~~~

Upstream-Status: Pending

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 src/xmlrpc_server_abyss.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/xmlrpc_server_abyss.c b/src/xmlrpc_server_abyss.c
index 2388652..3843f10 100644
--- a/src/xmlrpc_server_abyss.c
+++ b/src/xmlrpc_server_abyss.c
@@ -768,7 +768,7 @@ createServer(xmlrpc_env *                      const envP,
         ServerInit2(abyssServerP, &error);
 
         if (error) {
-            xmlrpc_faultf(envP, error);
+            xmlrpc_faultf(envP, "%s", error);
             xmlrpc_strfree(error);
         }
     }
-- 
2.8.1

